src/poker.js

```diff
@@ -1,5 +1,43 @@
+const cardValue = { 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8, 9: 9, 10: 10, J: 11, Q: 12, K: 13, A: 14 }
+
```

Comment:
I should incorporate more objects in my code... it looks neater, rather than creating another method. Cool!